feat: allow airplane mode control

NyCodeGHG opened the pull request:
## Proposed Changes

add a new command for controlling the airplane mode of android devices.

## Testing

ran the following flow on android 27 and 28 emulators:
```yaml
appId: org.wikipedia
---
- launchApp
- setAirplaneMode: disable
- setAirplaneMode: enable
```

axelniklasson commented:
@NyCodeGHG Nice 🎉 Let's update it to be a single command `- toggleAirplaneMode` instead and then we'll move forward with a formal review to get it merged. Thanks!

axelniklasson reviewed (changes requested):
https://github.com/mobile-dev-inc/maestro/pull/1672#issuecomment-2125030154

NyCodeGHG commented:
@axelniklasson hi, thank you for the review.
When I implemented this feature I had another option to toggle the airplane mode, but I removed it as it doesn't explicitly define the desired behavior. 
I can add another option to toggle if you want, but I'd prefer to keep the enable and disable options for those who want to be explicit.

axelniklasson commented:
> @axelniklasson hi, thank you for the review. When I implemented this feature I had another option to toggle the airplane mode, but I removed it as it doesn't explicitly define the desired behavior. I can add another option to toggle if you want, but I'd prefer to keep the enable and disable options for those who want to be explicit.

Hey, that's fair and a good use case to me. Let's keep the API as it is but also add `- toggleAirplaneMode` as a separate command

axelniklasson commented on maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt:
```diff
@@ -866,6 +867,28 @@ data class StopRecordingCommand(
 }
 
+enum class AirplaneValue {
+    @JsonProperty("enable")
```
```suggestion
    @JsonProperty("enabled")
```

axelniklasson commented on maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt:
```diff
@@ -866,6 +867,28 @@ data class StopRecordingCommand(
+enum class AirplaneValue {
+    @JsonProperty("enable")
+    Enable,
+    @JsonProperty("disable")
```
```suggestion
    @JsonProperty("disabled")
```

axelniklasson commented on maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt:
```diff
@@ -866,6 +867,38 @@ data class StopRecordingCommand(
+) : Command {
+    override fun description(): String {
+        return when (value) {
+            AirplaneValue.Enable -> "Enable Airplane mode"
+            AirplaneValue.Disable -> "Disable Airplane mode"
```
```suggestion
            AirplaneValue.Enable -> "Enable airplane mode"
            AirplaneValue.Disable -> "Disable airplane mode"
```

axelniklasson reviewed (approved):
Nice, this works well for me! Can you also raise a PR in https://github.com/mobile-dev-inc/maestro-docs as well as adding another test case to the integration tests? Thanks!

NyCodeGHG commented:
@axelniklasson 
I'm not sure on the integration test case
Can you elaborate?

axelniklasson commented:
Thanks @NyCodeGHG! Merging this now but please do also raise a PR in https://github.com/mobile-dev-inc/maestro-docs documenting the new commands - we'll merge it when we make the next release 👍 

Thanks again for your contribution!

NyCodeGHG commented:
will do!
